### PR TITLE
Feature/3017 activate edit and delete icons

### PR DESCRIPTION
**Pull request by Harshvardhan80**
Targets #3017 

### Description
Activate the edit icon
Activate delete icon

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: database/migrations/2023_04_17_113950_create_office_locations_table.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('office_locations', function (Blueprint $table) {
```
Comment (rahulpanwar16) [inline database/migrations/2023_04_17_113950_create_office_locations_table.php:36]:
```suggestion
        Schema::dropIfExists('office_locations');
 {
```

**Review by rahulpanwar16 — CHANGES_REQUESTED** [review]
**Testing Feedback** 

When user clicks on edit button the modal pops in and when user clicks on outside the modal, the modal closes and leave a blank page. Please fix it. 
Screenshot (95)

**Comment by rahulpanwar16** [comment]
@Harshvardhan80  Can you put ```assignees``` to yourself.

File: Modules/Operations/Resources/views/office-location/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+@extends('operations::layouts.master')
+
+@section('content')
+<div class="container">
+    <div>
+        <br>
+        <form action="{{ route('office-location.update', $centre->id) }}" method="POST">
```
Comment (rahulpanwar16) [inline Modules/Operations/Resources/views/office-location/edit.blade.php:7]: Please bring back the modal as it was earlier and move js code to app.js

Comment (Harshvardhan80) [reply Modules/Operations/Resources/views/office-location/edit.blade.php:7]: Javascript used or not
Screenshot_2023-04-26-12-40-38-40_40deb401b9ffe8e1df2f1cc5ba480b12

File: database/migrations/2023_04_08_181613_create_office_locations_table.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?php
+
```
Comment (GauravGusain98) [inline database/migrations/2023_04_08_181613_create_office_locations_table.php:2]: Why this change is still present in this PR? 

Did you make the new branch over the old branch. While starting a new task make sure to switch back to master branch and take latest pull and then create new branch from there. 

Now just take the pull of master branch in this branch and push it

File: Modules/Operations/Http/Controllers/OperationsController.php [hunk 4]
```diff
@@ -24,42 +30,44 @@ public function create()
...
+
+        return redirect()->route('office-location.index');
     }
 
     /**
      * Remove the specified resource from storage.
-     * @param int $id
      */
-    public function destroy($id)
+    public function destroy($id, OperationService $service)
```
Comment (GauravGusain98) [inline Modules/Operations/Http/Controllers/OperationsController.php:67]: You should use Model Class in the attribute that way you can directly get the object of the that Class corresponding to that id. 

This concept is called route-model binding. You can read about it form [here](https://laravel.com/docs/8.x/routing#route-model-binding)

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

File: database/migrations/2023_04_08_181613_create_office_locations_table.php (hunk 3 above)
Comment (Harshvardhan80) [reply database/migrations/2023_04_08_181613_create_office_locations_table.php:2]: @GauravGusain98    Sir my previous PR is still not merged in the master so even after taking the pull from master, my old files are still showing there

File: Modules/Operations/Entities/OfficeLocation.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+namespace Modules\Operations\Entities;
+
+use Illuminate\Database\Eloquent\Model;
+use Modules\User\Entities\User;
+
+class OfficeLocation extends Model
+{
+    protected $guarded = [];
+
+    public function centre_head()
```
Comment (GauravGusain98) [inline Modules/Operations/Entities/OfficeLocation.php:12]: Fix naming convention of the function

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
please check the comment

**Pull request merged**